examples/js/controls/TransformControls.js

```diff
@@ -903,7 +913,7 @@
 
 		function onPointerMove( event ) {
 
-			if ( scope.object === undefined || scope.axis === null || _dragging === false || ( event.button !== undefined && event.button !== 0 ) ) return;
```

Comment:
Which side effect can be produced by this? The problem is, that on the pointermove event I will receive an event.button=0, so it will return immediately without handling the event.